### ISSUE TITLE
Game uses deprecated GLSL syntax in some cases

### ISSUE THREAD

**Issue by stephengtuggy** [vegastrike/Assets-Masters#44]
I see lots of warnings about it in the logs when I run the game on my laptop that has an AMD GPU. I will try to find an example and attach it shortly.

**Comment by stephengtuggy** [comment]
Assigning this to myself, as I need to add the warning messages from my AMD machine.

**Comment by BenjamenMeyer** [comment]
Yes we're probably out of date on some of that stuff. We need to overhaul the engine to use Vulkan and more advanced GL functionality. But that's going to require a lot of learning and effort on our parts.
